src/js/background/background.js

```diff
@@ -94,6 +94,40 @@ async function patchMaskInfo(method = "PATCH", id, data, opts=null) {
   return await response.json();
 }
 
+async function postReportWebcompatIssue(description) {
+  const { relayApiSource } = await browser.storage.local.get("relayApiSource");  
+  if (!relayApiSource) {
+    return;
+  }
+  const headers = await createNewHeadersObject({auth: true});
+  const reportWebCompatResponse = `${relayApiSource}/report_webcompat_issue`;
```

Comment:
This variable name seems misleading - it contains a URL (I think?) rather than a response?